test/acl.test.js

```diff
@@ -126,52 +134,55 @@ describe('security ACLs', function() {
         ACL.checkPermission(ACL.USER, 'u001', 'User', 'name', ACL.ALL, function(err, perm) {
           assert(perm.permission === ACL.DENY);
         });
-
       });
-
     });
-
   });
 
   it('should allow access to models by exception', function() {
-    ACL.create({principalType: ACL.USER, principalId: 'u001', model: 'testModel', property: ACL.ALL,
-      accessType: ACL.ALL, permission: ACL.DENY}, function(err, acl) {
-
-      ACL.create({principalType: ACL.USER, principalId: 'u001', model: 'testModel', property: ACL.ALL,
-        accessType: ACL.READ, permission: ACL.ALLOW}, function(err, acl) {
-
-        ACL.create({principalType: ACL.USER, principalId: 'u002', model: 'testModel', property: ACL.ALL,
-          accessType: ACL.EXECUTE, permission: ACL.ALLOW}, function(err, acl) {
-
-          ACL.checkPermission(ACL.USER, 'u001', 'testModel', 'name', ACL.READ, function(err, perm) {
+    ACL.create({
+      principalType: ACL.USER, principalId: 'u001', model: 'testModel', property: ACL.ALL,
+      accessType: ACL.ALL, permission: ACL.DENY,
+    }, function(err, acl) {
+      ACL.create({
+        principalType: ACL.USER, principalId: 'u001', model: 'testModel', property: ACL.ALL,
+        accessType: ACL.READ, permission: ACL.ALLOW,
+      }, function(err, acl) {
+        ACL.create({
+          principalType: ACL.USER, principalId: 'u002', model: 'testModel', property: ACL.ALL,
+          accessType: ACL.EXECUTE, permission: ACL.ALLOW,
+        }, function(err, acl) {
+          ACL.checkPermission(ACL.USER, 'u001', 'testModel', 'name', ACL.READ,
+          function(err, perm) {
```

Comment:
I don't mind, as long as we have a convention for wrapping long lines. I find it's done in too many different ways, let's use your precribed way since it passes linting anyways.